internal/cmd/base/servers.go

```diff
@@ -51,6 +54,9 @@ type Server struct {
 	CombineLogs bool
 	LogLevel    hclog.Level
 
+	SerializationLock *sync.Mutex
```

Comment:
I get after reading more of the PR that this is really "serialization to stderr" but initially it seemed like it was related to serialization of the config, e.g. if we get a SIGHUP. If you can think of a better name it might be clearer.

Reply:
Renamed to StderrLock.  ty!